src/main/kotlin/ro/hibyte/betting/controller/BetController.kt

```diff
@@ -0,0 +1,44 @@
+package ro.hibyte.betting.controller
+
+import org.springframework.web.bind.annotation.*
+import ro.hibyte.betting.dto.BetDTO
+import ro.hibyte.betting.dto.UserProfileDTO
+import ro.hibyte.betting.entity.Bet
+import ro.hibyte.betting.entity.UserProfile
+import ro.hibyte.betting.service.BetService
+import ro.hibyte.betting.service.UserProfileService
+
+@CrossOrigin(origins = ["http://localhost:4200"])
+@RestController
+@RequestMapping("/api/bets")
```

Comment:
It's generally a good practice to also include a version in the API's URL (something like `/api/v1/bets`).